internal/printer/printer.go

```diff
@@ -102,27 +97,24 @@ const printerKey contextKey = 0
 
 func (pr *printer) PrintPackage(p *pkg.Pkg, leadingNewline bool) {
 	if leadingNewline {
-		fmt.Fprint(pr.outStream, "\n")
+		fmt.Fprint(pr.errStream, "\n")
```

Comment:
Similar to my comment above, this is being used by packaging. We should make sure this is changed consistently across all kpt commands.

Reply:
So, the package results logs are also printed to stderr. What more changes should be done for pkg group ?

Reply:
I think the printing is mostly done through the printer for packaging. I think the exception might be the output from git that is piped to the output. It might already end up on stderr, but I'm not sure.